src/main/java/com/laytonsmith/core/functions/Echoes.java

```diff
@@ -836,6 +842,19 @@ public Mixed exec(Target t, Environment environment, Mixed... args) throws Confi
 							b.append(color.exec(t, environment, new CString(c, t)));
 							i += sl;
 						} else {
+							if (c.equals('#')) {
+								try {
+									String subsequence2 = stext.substring(i + sl, i + sl + 7);
+									if (subsequence2.matches("(?i)^#[0-9A-F]{6}$")) {
```

Comment:
Build error:
`src\main\java\com\laytonsmith\core\functions\Echoes.java:848: Whitespace between if/for/while and ( is not allowed. [RegexpSingleline]`.